apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/resources/FISS.ndjson

```diff
@@ -0,0 +1,100 @@
+{"changeType":"CHANGE_TYPE_UPDATE","fissClaim":{"dcn":"6796658","hicNo":"672783318942","totalChargeAmount":"928.08","principleDiag":"xg","fissProcCodes":[{"procCd":"xg","procDt":"2021-07-11"},{"procCd":"qvwkg","procFlag":"ptwt","procDt":"2021-06-18"},{"procCd":"kstvc"},{"procCd":"znjkq","procFlag":"bg","procDt":"2021-07-16"},{"procCd":"bxng","procDt":"2021-02-13"},{"procCd":"zd","procDt":"2021-01-31"}],"pracLocState":"cg","fissDiagCodes":[{"diagCd2":"wcwp","bitFlags":"fng","diagPoaIndUnrecognized":"r"}],"currStatusUnrecognized":"k","currLoc1Unrecognized":"b","currLoc2Unrecognized":"m"}}
```

Comment:
just double checking, but this is all randomly generated data right?

Reply:
Yep it's random data generated using the mock server code.